src/SqlPersistence/Outbox/OutboxCommandBuilder.cs

```diff
@@ -20,11 +20,85 @@ public static OutboxCommands Build(string tablePrefix, string schema, SqlVariant
                 case SqlVariant.MySql:
                     tableName = $"`{tablePrefix}OutboxData`";
                     break;
+                case SqlVariant.Oracle:
+                    tableName = $"\"{tablePrefix}OD\"";
```

Comment:
I got burned in the acceptance tests by having the double quotes in here, because then `select from "TableName"` doesn't work because of casing, where `select from TableName` is understood just fine.